portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,12 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final String WELCOME_MESSAGE = "Welcome to my portfolio site! Hello Alisha!";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String message = WELCOME_MESSAGE;
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Alisha!</h1>");
+    response.getWriter().println(message);
```

Comment:
Change to: `response.getWriter().println(WELCOME_MESSAGE);`

The static variable can be called in any function in the class, just as if it was a local variable to the method. This also means you do not need your local `message` variable.

Reply:
ah gotcha!